api/interestr/api/models.py

```diff
@@ -46,11 +46,11 @@ def __str__(self):
 class Group(BaseModel):
     name = models.CharField(max_length=40)
     description = models.TextField(blank=True, default='')
-    location = models.TextField(blank = True, default='')
-    tags = models.TextField(blank = True, default='') #TODO change to list
-    #tags = models.ManyToManyField(Tag, related_name='tags')
```

Comment:
We should add this instead of the current one which is just a text field.

Reply:
👍